examples/multiomics_mogonet/configs/MOGONET_BRCA.yaml

```diff
@@ -0,0 +1,13 @@
+DATASET:
+  NAME: "TCGA_BRCA"
+  URL: "https://github.com/pykale/data/raw/main/multiomics/TCGA_BRCA.zip"
+  NUM_MODALITIES: 3
+  NUM_CLASSES: 5
+
+SOLVER:
+  MAX_EPOCHS_PRETRAIN: 5 # For quick testing
+  MAX_EPOCHS: 10  # For quick testing
```

Comment:
Should the file name reflect this?

Reply:
I have renamed the file to "MOGONET_BRCA_quick_test.yaml".